generators/server/templates/pom.xml.ejs

```diff
@@ -88,7 +88,7 @@
         <profile.swagger />
 
         <!-- Dependency versions -->
-        <jhipster-dependencies.version>2.0.3</jhipster-dependencies.version>
+        <jhipster-dependencies.version>2.0.5-SNAPSHOT</jhipster-dependencies.version>
```

Comment:
I don't think those can work, our snapshot repo on Bintray isn't configured. I'll first do a 2.0.5 release of jhispter-framework etc, and then update this.

Reply:
Sorry, I wasn't sure if I remembered correctly -- but didn't we eventually figure out that we can publish snapshots via sonatype, previously?

Reply:
Yes but for this we need to have the bintray repository configured in the final pom.xml - and I don't think this is the case anymore
Not a big problem: I'm testing locally then pushing once it builds